content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -739,6 +739,106 @@ query.iter().for_each(|mut component| {
 
 These abstractions were introduced in [#6404](https://github.com/bevyengine/bevy/pull/6404), [#7381](https://github.com/bevyengine/bevy/pull/7381) and [#7568](https://github.com/bevyengine/bevy/pull/7568).
 
+## Reflection Paths: Enums and Tuples
+
+<div class="release-feature-authors">authors: @MrGVSV</div>
+
+Bevy's "reflection paths" enable navigating Rust values using a simple (and dynamic) string syntax. **Bevy 0.10** expands this system by adding support for tuples and enums in reflect paths:
+
+```rust
+#[derive(Reflect)]
+struct MyStruct {
+  data: Data,
+  some_tuple: (u32, u32),
+}
+
+#[derive(Reflect)]
+struct Data {
+  Foo(u32, u32),
+  Bar(bool)
+}
+
+let x = MyStruct {
+  data: Data::Foo(123),
+  some_tuple: (10, 20),
+};
+
+assert_eq!(*x.path::<u32>("data.1").unwrap(), 123);
+assert_eq!(*x.path::<u32>("some_tuple.0").unwrap(), 10);
+```
+
+## Pre-Parsed Reflection Paths
+
+<div class="release-feature-authors">authors: @MrGVSV</div>
```

Comment:
This PR was just a rebase of @james7132's PR, so they should probably be listed as an author as well :)

Reply:
```suggestion
<div class="release-feature-authors">authors: @MrGVSV, @james7132 </div>
```